Diary/src/main/java/com/microsoft/device/display/samples/diary/FileOps.kt

```diff
@@ -0,0 +1,70 @@
+package com.microsoft.device.display.samples.diary
+
+import android.Manifest
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Build
+import android.os.Environment
+import android.util.Log
+import androidx.core.content.ContextCompat
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileReader
+import java.io.FileWriter
+import java.io.IOException
+
+class FileOps {
+
+    fun readDayFile(fileName: String, rootDir: File): String {
+        val file = File(rootDir, "/$fileName")
+
+        if (!file.exists()) {
+            return "Let's start writing for the day"
```

Comment:
Would you please move the string into the resource file?

Reply:
Hey joy, I tried to figure it out, but it's not letting me use stringResource inside the Fileoperation file

Reply:
updated it with stringResource